Project.toml

```diff
@@ -13,6 +13,7 @@ Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Nemo = "2edaba10-b0f1-5616-af89-8c11ac63239a"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+lib4ti2_jll = "1493ae25-0f90-5c0e-a06c-8c5077d6d66f"
```

Comment:
Don't you need a version below? Also, is this how jll's are dealt with? Perhaps @fingolfin can enlighten, as I've not used them.

Reply:
JLLs are regular packages, so yeah, that's how you deal with them (well, at least with regards to `Project.toml`). You can specify a compat version, but don't have to (again, just like with other packages). It wouldn't hurt to specify one, I guess (and the Julia package registrator might complain if we don't have one), but 4ti2 is not likely to receive major changes anymore, so there is no reasonable expectations of "breaking updates" or whatever :-)